packages/devextreme/js/__internal/utils/version.ts

```diff
@@ -25,6 +29,45 @@ export function parseVersion(version: string): Version {
   };
 }
 
+export function reportDependentVersion(dependentName: string, version: string): void {
```

Comment:
As for me, I mostly prefer the verb "assert", as it's very common in checking prerequisites.

Maybe `assertDevExtremeVersion` instead of shorter `assertVersion`